### DISCUSSION TITLE
Sequentially played videos within one observation - can a time gap be set between two videos?

### DISCUSSION THREAD

**Discussion by clearwingMSV** [olivierfriard/BORIS#570] [Q&A] [answered]
Hi,

I have several videos within the same observation that have to be analysed sequentially. I select the same player for the video files so events coded at time tx in the media file queued as second in the playlist are cored as happening at time t1 + tx (where t1 is the duration of the first media file). Is there any way to add a "time gap" between the first and second video so that e.g. events coded in the second video would be at time t1 + 15 seconds + tx?

I am in need of this feature because while I was recording a behaviour, sometimes an error occured in the camera and the video was unexpectedly stopped and then re-initiated asap, during this time the animal was engaged in a behaviour that I would like to code. Ideally I'd be able to code this behaviour as staring in video 1, lasting 15 seconds between video 1 and 2, and then continuing for another x seconds in video 2. Obviously all further events should be coded including this time gap between video 1 & 2. Is this possible?

Thanks in advance for any tips/suggestions.

**Comment by olivierfriard** [comment, marked as answer]
Hi,
no, it is not possible

**Comment by olivierfriard** [comment]
I am working on an option for using the Media create date/time tag from the video metadata to be used as offset for the coding timestamp. Check if your videos have this tag and stay in touch!

**Comment by clearwingMSV** [comment]
Thank you very much for the tip, Olivier, I will be glad to keep up-to-date with any progress on this issue!
